docs/source/using-discovery/usingslurm.rst

```diff
@@ -3,51 +3,95 @@
+* Check the cluster state, along with that of specific nodes
+* Gain knowledge of various best practices when using SLURM
+
+This page is specific for using Slurm on Discovery. For more in-depth and general information, please refer to the official `Slurm documentation`_.
+
+
+Slurm CLI
+==============
+List of typical Slurm commands for working on Discovery.
+
+These functions are fully documented for the version of Slurm installed on in the manual pages. To get the manual pages for the Slurm functions, run ``man <SLURM_command>`` on the HPC. This command lists all the parameters and keywords that you can use with the given Slurm commands. You can also run ``<SLURM_command> --help``, which will list the help page for the Slurm function. These two give you the full possibilities of what these functions can do. The following sections are helpful/commonly used Slurm commands that are good to know for different ticket requests and common checks on the cluster.
 
 .. important::
-   Slurm commands have numerous options to help your jobs run efficiently by requesting specific resources. Options also usually have both short and verbose versions, such as
-   ``--nodes`` and ``-n`` (both mean the same thing). The examples in this documentation all use the
-   verbose version of the options for clarity, but you can use the short version if you prefer. For example, ``srun -p short -N 1 -n 1`` means the exact same thing as ``srun --partition=short --nodes=1 --ntasks=1``
-   Refer to the official Slurm documentation to get in-depth information about these commands and their options: `Slurm documentation website <https://slurm.schedmd.com/archive/slurm-17.11.6/srun.html>`_.
+   Slurm commands have numerous options to help your jobs run efficiently by requesting specific resources. Options usually have both short and verbose versions, such as ``--nodes`` and ``-n`` (both mean the same thing). The examples in this documentation all use the verbose version of the options for clarity, but you can use the short version if you prefer. For example, ``srun -p short -N 1 -n 1`` means the same thing as ``srun --partition=short --nodes=1 --ntasks=1``
+
+Controlling jobs
+----------------
+``scontrol hold <jobid>`` Place a hold on a pending job
+
+``scontrol release <jobid>`` Release a held job
+
+``scontrol requeue <jobid>`` Requeue a completed, failed, or cancelled job
+
+Job reporting
+-------------
+``sacct <options>`` Display job accounting information
+
+``sreport <options>`` Generate reports about cluster utilization and job statistics
 
-Viewing Cluster Information
+Advanced features
+------------------
+``sprio <options>`` Show the priority of jobs and job steps
+
+``sburst <options>`` Show the state of all burst buffer pools
+
+**NOTE:** Each command's options and functions may vary depending on the Slurm version and configuration. We recommend consulting the Slurm documentation for more information on the full range of available commands and their usage.
```

Comment:
Do we want to add the link to the Slurm documentation? I know we have it in a few other places, but wouldn't hurt to add.